app/helpers/application_helper.rb

```diff
@@ -13,8 +13,20 @@ def divide_in_chunks(arr, chunk_size)
     chunks
   end
 
-  def get_img_url(video)
-    video.img ? "/tmp/#{video.img}" : 'http://dummyimage.com/665x375/000000/00a2ff'
+  def get_large_cover_url(video)
```

Comment:
hmmm didn't know that there is ```large_cover_url``` as well... there are so many methods added by Carrierview

```
2.1.7 :005 > video.methods.grep /large/
 => [:large_cover, :large_cover=, :remote_large_cover_url=, :remove_large_cover!, :get_large_cover_url, :large_cover?, :large_cover_url, :large_cover_cache, :large_cover_cache=, :remote_large_cover_url, :remove_large_cover, :remove_large_cover=, :remove_large_cover?, :store_large_cover!, :large_cover_integrity_error, :large_cover_processing_error, :large_cover_download_error, :write_large_cover_identifier, :large_cover_identifier, :store_previous_model_for_large_cover, :find_previous_model_for_large_cover, :remove_previously_stored_large_cover, :mark_remove_large_cover_false] 
```

I moved those methods into the model. Nevertheless, the method is not working because the dummy image is returned even if there is an image. My implementation:

```
  def large_cover_url
    large_cover.url || 'http://dummyimage.com/665x375/000000/00a2ff'
  end
```